mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -248,18 +248,18 @@ class ALSModel private[ml] (
   @Since("1.3.0")
   def setPredictionCol(value: String): this.type = set(predictionCol, value)
 
+  private val predict = udf { (userFeatures: Seq[Float], itemFeatures: Seq[Float]) =>
```

Comment:
They are switched in the case of `recommendForAllItems` so is that not less clear?